src/main/resources/site/parts/highchart/highchart.html

```diff
@@ -30,8 +30,7 @@ <h3 class="ssb-title" data-th-if="${highchart.config.title.text}" data-th-text="
       <div class="footnote" data-th-if="${highchart.footnoteText}" data-th-text="${highchart.footnoteText}"></div>
       <div class="d-none highcharts-table invisible" data-th-utext="${highchart.tableData}"></div>
     </div>
-    <div data-th-if="${highchart.creditsEnabled}" class="col-12 mt-4 mt-md-5 highcharts-source">
-      <h5 class="mb-0" data-th-text="${sourceText}">Kilde:</h5>
+    <div data-th-if="${highchart.creditsEnabled}" class="col-12 highcharts-source">
       <a class="ssb-link" data-th-href="${highchart.creditsHref}" data-th-text="${highchart.creditsText}"></a>
```

Comment:
Bør kanskje legge på kildefrasen her istedenfor i ts-filen siden koden for å legge til flere kilder også trenger sourceText :) Feks slik: data-th-text="${sourceText} + ' ' + ${highchart.creditsText}"